livemaps-api/APIFunctions/DeviceState.cs

```diff
@@ -38,10 +37,10 @@ public static class DeviceState
 
                 fileName = fileName.ToLower();
                
-                var devicestateref = container.GetBlockBlobReference(fileName);
+                var devicestateref = container.GetBlobClient(fileName);
                 using (var ms = new MemoryStream())
                 {
-                    await devicestateref.DownloadToStreamAsync(ms);
+                    await devicestateref.DownloadToAsync(ms);
```

Comment:
We should use DownloadContentAsync instead if we just want to get the blob as a string.  It's much simpler than all the stream code.  Here's a rough example of what we end up with:
```C#
BlobClient deviceStateBlob = container.GetBlobClient(fileName);
BlobDownloadResult result = await deviceStateBlob.DownloadContentAsync();
string deviceStateData = result.Content.ToString();
return new OkObjectResult(deviceStateData)
```

Reply:
Ditto for everything else here.